### ISSUE TITLE
No mouth

### ISSUE THREAD

**Issue by gingerling** [MycroftAI/skill-singing#9]
Hey would be great if mycroft's mouth would move while he sings, but he goes back to sleep

**Comment by penrods** [comment]
I agree!  It would technically be possible to do this, but would require a fair amount of manual effort.  The "singing" is not generated dynamically, so this will need someone to produce the viseme sequence manually and align the timing.

If anyone wants to undertake this, I'd be happy to provide advice on how to do it.

**Comment by gingerling** [comment]
Do you think this would be possible for an intelligent technically minded
NON-programmer to have a go at? It strikes me that potentially it's a bit
like stop-motion animation, which I have done before, I'm wondering how
repetitive the programming would be - if it's a case of you write the
beginning and end code and I fill in something which says "what
milliseconds, which mouth shape" for the entire song that is possibly
doable...
